src/main/java/it/pagopa/ecommerce/commons/domain/PaymentNotice.java

```diff
@@ -24,6 +24,8 @@ public record PaymentNotice(
 
         List<PaymentTransferInfo> transferList,
 
-        boolean isAllCCP
+        boolean isAllCCP,
+
+        CompanyName companyName
```

Comment:
Can you add these two to the javadocs?

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-commons/pull/194/commits/68fb423c4c5153aa274f61d4a06b31492b934f83